docs/onboarding_guide.mdx

```diff
@@ -7,11 +7,18 @@ import Image from "@theme/IdealImage";
 
 # Onboarding (In Draft)
 
-<div class="intro-logo">
-  <img src="/img/logo.svg" alt="logo" />
-</div>
-<br />
-<br />
+Thank you for your interesting in participating in the Standards Working Group! We're excited to have you contribute to this project.
 
-## Onboarding
+To get the most out of the working group we've created a few resources to get you up to speed: 
 
+## Read the Specification
+The specification forms the foundation of everything that this working group focuses on. We suggest that you read the SCI specification cover to cover (don't worry, it's not that long!). Not only will you be intimately familiar with this key document, but your fresh eyes will help identify gaps and limitations in the current specification. If there are areas that you have questions about and want to dig deeper, we suggest looking at the FAQ (forthcoming). If questions remain or there are ambiguities, this may be a perfect area that you can help us improve the specification. The most recent version of the specificaiton can be found here <link> (it might be wise to view the 'dev' branch to see the current draft version as well).
```

Comment:
```suggestion
The specification forms the foundation of everything that this working group focuses on. We suggest that you read the SCI specification cover to cover (don't worry, it's not that long!). Not only will you be intimately familiar with this key document, but your fresh eyes will help identify gaps and limitations in the current specification. If there are areas that you have questions about and want to dig deeper, we suggest looking at the FAQ (forthcoming). If questions remain or there are ambiguities, this may be a perfect area that you can help us improve the specification. The most recent version of the specificaiton can be found [here](https://github.com/Green-Software-Foundation/software_carbon_intensity) (it might be wise to view the 'dev' branch to see the current draft version as well).
```